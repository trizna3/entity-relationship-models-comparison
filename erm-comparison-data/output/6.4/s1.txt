points-ways
pictures-pictures
trips-trips
users-users
