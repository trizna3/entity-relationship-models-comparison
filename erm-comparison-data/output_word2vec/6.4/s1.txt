trips-trips
points-ways
pictures-pictures
users-users
