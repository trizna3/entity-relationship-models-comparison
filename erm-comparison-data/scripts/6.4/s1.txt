entity set users {
	"name"
	"passwords"
};

entity set trips {
	"descriptions"
};

entity set pictures {
	"x"
	"y"
	"title"
};

entity set ways {
	"x"
	"y"
};

entity set ratings {
	"rate"
};

association between 
{
	users ONE 
	trips MANY
}{};

association between 
{
	users ONE 
	pictures MANY
}{};

association between 
{
	pictures MANY 
	trips ONE 
}{};

association between 
{
	trips ONE 
	ways MANY
}{};

association between 
{
	trips ONE 
	ratings MANY
}{};

