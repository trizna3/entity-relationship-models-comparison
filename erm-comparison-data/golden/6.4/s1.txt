users-users
trips-trips
pictures-pictures
points-ways